src/clj_zeppelin/core.clj

```diff
@@ -68,8 +74,14 @@
 
   Returns the id of the imported note
   "
-  []
-  nil)
+  [notebook-api-url note_json_url]
```

Comment:
Given this approach (of downloading the notebook data), you might want to

- try downloading the notebook data
- if that fails, create a different error stating that the note data is not accessible
- if not, continue to import.

Reply:
Import notebook issue is resolved.